packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelSummary.tsx

```diff
@@ -274,28 +299,68 @@ export function TransferPanelSummary({
 }: TransferPanelSummaryProps) {
   const { status, estimatedL1GasFees, estimatedL2GasFees } = gasSummary
 
-  const { app } = useAppState()
+  const {
+    app: { isDepositMode }
+  } = useAppState()
   const { ethToUSD } = useETHPrice()
   const { l1, l2 } = useNetworksAndSigners()
   const { parentLayer, layer } = useChainLayers()
 
   const nativeCurrency = useNativeCurrency({ provider: l2.provider })
   const parentChainNativeCurrency = useNativeCurrency({ provider: l1.provider })
 
+  const parentLayerGasFeeTooltipContent = useMemo(() => {
```

Comment:
The entire switch statement along with the `layerToGasFeeTooltip` object is a bit hard to read and follow. Would this make things easier?
```
const l2FeesContent = "L2 fees are collected by the chain to cover costs of execution. This is an estimated fee, if the true fee is lower, you'll be refunded."

const parentLayerGasFeeTooltipContent = useMemo(() => {
   const { isEthereum } = isNetwork(l1.network.id)

   if (isDepositMode) {
      if (isEthereum) {
        return 'L1 fees go to Ethereum Validators.'
      }
      // L2 network, because Orbit chains can't be parent chains yet
      return l2FeesContent
   }

   // withdrawals
   if (isEthereum) {
      return l2FeesContent
   }
   // L2 network, because Orbit chains can't be parent chains yet
   return 'A small L2 calldata fee is paid for including this Orbit transaction in the sequencer batch to be posted on L2.'
}, [])
```

And then similarly for `layerGasFeeTooltipContent`

Reply:
i cleaned it up as the UI changes are now diff than what it was